source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml

```diff
@@ -256,9 +267,22 @@ content: |
         .. tab::
            :tabid: nodejs
 
-           .. note::
+           .. code-block:: javascript
+
+              const encryption = new ClientEncryption(client, {
+                  keyVaultNamespace,
+                  kmsProviders
+              });
+              const key = await encryption.createDataKey('azure', {
```

Comment:
Question: are we using single or double quotes in the Node.js data/code examples? We should stay consistent whichever we choose. This code example uses single quotes while the data example on ll. 122-124 of Azure KMS use double quotes as well as your changes for GCP.

Reply:
Thanks for pointing out this oversight. I added some rules to the companion project to ensure consistent formatting in the future and double quotes are now standard, so I've gone with double quotes here.